include/seqan3/alphabet/concept_pre.hpp

```diff
@@ -221,6 +221,42 @@ using underlying_char_t = typename underlying_char<alphabet_type>::type;
 
 //!\}
 
+// ------------------------------------------------------------------
+// seqan3::rna_structure_concept
+// ------------------------------------------------------------------
+
+/*!\name Requirements for seqan3::rna_structure_concept
+ * \brief You can expect these functions on all types that implement seqan3::rna_structure_concept.
+ * \{
+ */
+/*!\brief Metafunction that indicates whether an alphabet can handle pseudoknots. [value metafunction base template]
+ * \tparam alphabet_type Must satisfy seqan3::rna_structure_concept.
```

Comment:
Strictly speaking it is the other way around: this property is prequisite for satisfying the seqan3::rna_structure_concept!